packaging/build/macos.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eux
+
+python -V
+pip install -U setuptools wheel pip Cython delocate
+python setup.py bdist_wheel
```

Comment:
It's recommended to use `pip wheel` or better [`python -m build`](https://github.com/pypa/build) instead of hitting an implementation detail